libcaf_io/caf/io/basp/routing_table.hpp

```diff
@@ -126,16 +99,11 @@ class routing_table {
     return std::forward<Fallback>(x);
   }
 
-  using node_id_set = std::unordered_set<node_id>;
-
-  using indirect_entries = std::unordered_map<node_id,      // dest
-                                              node_id_set>; // hop
-
   abstract_broker* parent_;
   std::unordered_map<endpoint_handle, node_id> direct_by_hdl_;
+  // TODO: Do we need a list as a second argument as there could be
```

Comment:
For now I think restricting to a single path is the only sane option. A list wouldn't do anyways, since we would have to specify what technologies are preferred (e.g. `Infiniband > TCP > UDP`).

"The first path wins" seems like a reasonable strategy for now. We can expect users to make intelligent deployment decisions and not mix UDP and TCP, for example.

Reply:
Sounds reasonable.